src/app/GitCommands/CommitDataManager.cs

```diff
@@ -73,7 +73,7 @@ public void UpdateBody(CommitData commitData, bool appendNotesOnly, out string?
         /// <inheritdoc />
         public CommitData? GetCommitData(string commitId, bool includeNotes = false)
         {
-            GitRevision revision = new RevisionReader(GetModule(), allBodies: true).GetRevision(commitId, hasNotes: includeNotes, cancellationToken: default);
+            GitRevision? revision = new RevisionReader(GetModule(), allBodies: true).GetRevision(commitId, hasNotes: includeNotes, throwOnError: false, cancellationToken: default);
```

Comment:
I have seen error thrown here for WSL, that seem to be a WSL issue when some commands are aborted (by Windows?). The exceptions should not occur normally, it is annoying to see them.
No real harm in hiding errors here (I do not see a reason for the exception to occur normally).